### PR TITLE
Add suport for freebsd package resource

### PR DESCRIPTION
**Pull request by fzipi**
## Description

This PR adds support for testing using package resources in FreeBSD.

## Related Issue

Created #4788.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New content (non-breaking change)
- [ ] Breaking change (a content change which would break existing functionality or processes)

## Checklist:

- [ x ] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by zenspider** [comment]
The failures on BK have nothing to do with your PR and everything to do with rubygems breaking things in the latest version. Ignore that for now. I have a PR coming shortly that should fix things. Rebasing against master once that goes in should clear this up.

**Comment by zenspider** [comment]
Merged... Please rebase against master and force push.

File: lib/inspec/resources/package.rb [hunk 1]
```diff
@@ -217,7 +219,7 @@ def info(package_name)
       }
     rescue JSON::ParserError => e
       raise Inspec::Exceptions::ResourceFailed,
-            "Failed to parse JSON from `brew` command. Error: #{e}"
+              "Failed to parse JSON from `brew` command. Error: #{e}"
```
Comment (zenspider) [inline lib/inspec/resources/package.rb]: Please don't reformat on functional PRs.

**Review by zenspider — COMMENTED** [review]
Minor nit (not need to fix unless you want to). Needs a rebase against master for proper CI. After that goes green, this should be good to go.

File: lib/inspec/resources/package.rb (hunk 1 above)
Comment (fzipi) [reply lib/inspec/resources/package.rb]: Weird, shouldn't have messed with that :thinking: . Anyway, removed change and pushed again.

**Review by miah — APPROVED** [review]

**Pull request merged**